azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_gcp_terraform.go

```diff
@@ -0,0 +1,81 @@
+package serviceendpoint
+
+import (
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/v7/serviceendpoint"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/converter"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/tfhelper"
+)
+
+// ResourceServiceEndpointGcp schema and implementation for gcp service endpoint resource
+func ResourceServiceEndpointGcp() *schema.Resource {
+	r := genBaseServiceEndpointResource(flattenServiceEndpointGcp, expandServiceEndpointGcp)
+	r.Schema["client_email"] = &schema.Schema{
```

Comment:
can we simplify the name to `email`?

Reply:
We could, but I kept it that way since that's what shows up in the ADO GUI and also in the GCP keyfile json.